spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -4807,6 +4807,8 @@ You can further tune how `RestClient` is configured, as shown in the following e
 	spring.elasticsearch.rest.uris=https://search.example.com:9200
 	spring.elasticsearch.rest.username=user
 	spring.elasticsearch.rest.password=secret
+	spring.elasticsearch.rest.connection-timeout=10s
```

Comment:
I don't think that's necessary to list all properties. The `jest` doc has a reference to `read-timeout` so let's align to that only.